src/App.js

```diff
@@ -1,6 +1,8 @@
 import './App.css';
 import Home from './components/HomePage/Home';
 import { Routes, Route } from "react-router-dom";
+import Login from './components/Login/Login';
+import SignUp from './components/SignUp/SignUp';
 import Navbar from './components/Navbar/Navbar';
```

Comment:
suggestion: create index file in components folder and export all components from it